test-manual/.env.development

```
@@ -0,0 +1,6 @@
+﻿DEBUG=true
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)